ui/app/selectors/custom-gas.js

```diff
@@ -210,39 +210,52 @@ export function getRenderableBasicEstimateData (state, gasLimit) {
         safeLowWait,
         avgWait,
         fastWait,
+        fastest,
+        fastestWait,
       },
     },
   } = state
 
-  return [
-    {
-      gasEstimateType: GAS_ESTIMATE_TYPES.SLOW,
-      feeInPrimaryCurrency: getRenderableEthFee(safeLow, gasLimit),
-      feeInSecondaryCurrency: showFiat
-        ? getRenderableConvertedCurrencyFee(safeLow, gasLimit, currentCurrency, conversionRate)
-        : '',
-      timeEstimate: safeLowWait && getRenderableTimeEstimate(safeLowWait),
-      priceInHexWei: getGasPriceInHexWei(safeLow),
-    },
-    {
-      gasEstimateType: GAS_ESTIMATE_TYPES.AVERAGE,
-      feeInPrimaryCurrency: getRenderableEthFee(average, gasLimit),
-      feeInSecondaryCurrency: showFiat
-        ? getRenderableConvertedCurrencyFee(average, gasLimit, currentCurrency, conversionRate)
-        : '',
-      timeEstimate: avgWait && getRenderableTimeEstimate(avgWait),
-      priceInHexWei: getGasPriceInHexWei(average),
-    },
-    {
-      gasEstimateType: GAS_ESTIMATE_TYPES.FAST,
-      feeInPrimaryCurrency: getRenderableEthFee(fast, gasLimit),
-      feeInSecondaryCurrency: showFiat
-        ? getRenderableConvertedCurrencyFee(fast, gasLimit, currentCurrency, conversionRate)
-        : '',
-      timeEstimate: fastWait && getRenderableTimeEstimate(fastWait),
-      priceInHexWei: getGasPriceInHexWei(fast),
-    },
-  ]
+  const slowEstimatData = {
+    gasEstimateType: GAS_ESTIMATE_TYPES.SLOW,
+    feeInPrimaryCurrency: getRenderableEthFee(safeLow, gasLimit),
+    feeInSecondaryCurrency: showFiat
+      ? getRenderableConvertedCurrencyFee(safeLow, gasLimit, currentCurrency, conversionRate)
+      : '',
+    timeEstimate: safeLowWait && getRenderableTimeEstimate(safeLowWait),
+    priceInHexWei: getGasPriceInHexWei(safeLow),
+  }
+  const averageEstimateData = {
+    gasEstimateType: GAS_ESTIMATE_TYPES.AVERAGE,
+    feeInPrimaryCurrency: getRenderableEthFee(average, gasLimit),
+    feeInSecondaryCurrency: showFiat
+      ? getRenderableConvertedCurrencyFee(average, gasLimit, currentCurrency, conversionRate)
+      : '',
+    timeEstimate: avgWait && getRenderableTimeEstimate(avgWait),
+    priceInHexWei: getGasPriceInHexWei(average),
+  }
+  const fastEstimatData = {
```

Comment:
Typo here

```suggestion
  const fastEstimateData = {
```

And similarly below where this is being used

Reply:
Also for `slowEstimatData` - missing `e` there as well